packages/story-editor/src/components/floatingMenu/elements/fontSize.js

```diff
@@ -41,6 +42,10 @@ function FontSize() {
 
   const pushUpdate = useCallback(
     (update) => {
+      trackEvent('floating_menu', {
+        name: 'set_font_size',
```

Comment:
🤔  Does this need the element type?

Reply:
nope cuz it's always text. it's only included if there is more than 1 option for the element.